lib/active_model/serializer.rb

```diff
@@ -118,11 +122,21 @@ def strip_attribute(attr)
         attr
       end
 
+      def build_serializer_class_list(resource, options)
+        list = []
+        list << build_serializer_class(resource, options)
+        list << build_serializer_class(resource.class.name.demodulize, {})
```

Comment:
lookinng at 0-10-stable lib/active_model_serializers/lookup_chain.rb for how we solved this when it came up there looks like the same solution  more or less, so that's good :)

```ruby
    module_function

    def namespace_for(klass)
      klass.name.deconstantize
    end

    def resource_class_name(klass)
      klass.name.demodulize
    end

    def serializer_from_resource_name(name)
      "#{name}Serializer"
    end

    def serializer_from(klass)
      name = resource_class_name(klass)
      serializer_from_resource_name(name)
    end
```

and that in the serializer we called `serializer_class = lookup_chain.map(&:safe_constantize).find { |x| x && x < ActiveModel::Serializer }`

Reply:
I got some inspiration from v0.10 for sure, thou i am still a little unclear on how the find condition works out. How does `x < ActiveModel::Serializer` work? 

Also, is there a need for me to make any changes to replicate how things are done in v0.10?

Reply:
`x < ActiveModel::Serializer` is just one of those weird-looking Ruby things for asking if a thing is a subclass of the thing on the right

Reply:
I'm guessing the 0.10 solution would work here as well. Up to you if you want to make another PR after this is merged